object_manager.go

```diff
@@ -369,6 +375,61 @@ func (objMgr *ObjectManager) CreateEADefinition(eadef EADefinition) (*EADefiniti
 	return newEadef, err
 }
 
+func (objMgr *ObjectManager) CreateRecordHostWithoutDNS(recordName string, netview string, cidr string, ipAddr string, macAddress string, vmID string) (*RecordHost, error) {
+
+	ea := objMgr.getBasicEA(true)
+	ea["VM ID"] = "N/A"
+	if vmID != "" {
+		ea["VM ID"] = vmID
+	}
+
+	recordHostIpAddr := NewRecordHostIpv4Addr(RecordHostIpv4Addr{Mac: macAddress})
+
+	if ipAddr == "" {
+		recordHostIpAddr.Ipv4Addr = fmt.Sprintf("func:nextavailableip:%s,%s", cidr, netview)
+	} else {
+		recordHostIpAddr.Ipv4Addr = ipAddr
+	}
+	enableDNS := new(bool)
+	*enableDNS = false
+	recordHostIpAddrSlice := make([]RecordHostIpv4Addr, 0)
+	recordHostIpAddrSlice = append(recordHostIpAddrSlice, *recordHostIpAddr)
```

Comment:
will just this work  
slice1 := []RecordHostIpv4Addr{*recordHostIpAddr}

Reply:
Done,